docs/admin/network-plugins.md

```diff
@@ -39,7 +39,7 @@ The Linux-only kubenet plugin provides functionality similar to the `--configure
 
 The plugin requires a few things:
 
-* The standard CNI `bridge` and `host-local` plugins to be placed in `/opt/cni/bin`.
+* The standard CNI `bridge` and `host-local` plugins to be placed in `/opt/cni/bin`. Or specify `network-plugin-dir` to supply additional search path for CNI plugins.  
```

Comment:
can you elaborate on which takes precedence, and that all binaries are expected to be in one location (i.e both bridge and host-local in --network-plugin-dir)?

I would phrase it as something like: network-plugin-dir with the bridge and host-local, defaults to /opt/cni...